pyproject.toml

```diff
@@ -141,27 +141,42 @@ exclude_lines = [
     "@overload",
 ]
 
+# https://mypy.readthedocs.io/en/stable/config_file.html
 [tool.mypy]
-python_version = "3.10"
```

Comment:
This isn't particularly useful, what we really need is https://github.com/python/mypy/issues/12286